src/store/user/slice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface userState {
+  isLogged: boolean;
+  isAdmin: boolean;
+}
+
+const initialState: userState = {
+  isLogged: false,
+  isAdmin: false,
+};
+
+export const userSlice = createSlice({
+  name: 'user',
+  initialState,
+  reducers: {
+    signIn: (state, action: PayloadAction<boolean>) => {
+      state.isLogged = true;
+      state.isAdmin = action.payload;
+    },
+    signOut: state => {
+      state.isLogged = false;
+      state.isAdmin = false;
+    },
```

Comment:
We will use extraReducers and thunk actions for this
```suggestion
  reducers: {},
  extraReducers: builder => {
    builder.addCase(signIn.fulfilled, (state, { payload }) => {
      state.user = payload;
      state.isAuthorized = Boolean(payload);
    });
  },
```